src/ray/raylet/scheduling_resources.h

```diff
@@ -387,4 +387,20 @@ class SchedulingResources {
 
 }  // namespace ray
 
+namespace std {
+template <>
+struct hash<ray::raylet::ResourceSet> {
+  std::size_t operator()(ray::raylet::ResourceSet const &k) const {
+    // Combining hash functions, see https://stackoverflow.com/a/2595226
+    std::size_t seed = k.GetResourceMap().size();
+    for (auto &elem : k.GetResourceMap()) {
```

Comment:
Will this work no matter what the order of iteration over `k.GetResourceMap()` is?

Reply:
Oh, never mind, just noticed it's doing `^=` below :)